client/components/layout/templates/sidebar.jade

```diff
@@ -25,11 +25,11 @@
         include ../public/icons/layout_verticals.svg
         br
         | Verticals
-      a( href='/organizations'
-         class=(sd.URL.match('/organizations') ? 'is-active' : '') )
+      a( href='/brand-partners'
+         class=(sd.URL.match('/brand-partners') ? 'is-active' : '') )
         include ../public/icons/layout_organizations.svg
         br
-        | Orgs
+        | Brands
```

Comment:
Might not be the best icon to use -- let me know if you have a better one! @owendodd